orca/scripts/build_services.sh

```diff
@@ -64,7 +64,7 @@ function build_service_db {
 	write 1 "        - MARIADB_ROOT_HOST=localhost"
 	write 1 "        - MARIADB_ROOT_PASSWORD_FILE=/tmp/orca-secrets/mysql_root_password"
 	write 1 "        - MARIADB_USER=lportal"
-	write 1 "        - ORCA_DB_ADDRESSES=$(query_services db host_port 3306 true)"
+	write 1 "        - ORCA_DB_ADDRESSES=$(query_services db host_port 3306 false)"
```

Comment:
If I see correct, this variable is not used so we can remove.

Reply:
Fixed.